pycbc/inference/models/gaussian_noise.py

```diff
@@ -160,53 +165,25 @@ def __init__(self, variable_params, data, low_frequency_cutoff, psds=None,
                                                      d.delta_f, self._N)
             self._kmin[det] = kmin
             self._kmax[det] = kmax
+
         # store the psd segments
         self._psd_segments = {}
         if psds is not None:
             self.set_psd_segments(psds)
+
         # store the psds and calculate the inner product weight
         self._psds = {}
         self._weight = {}
         self._lognorm = {}
         self._det_lognls = {}
         self._whitened_data = {}
+
         # set the normalization state
         self._normalize = False
         self.normalize = normalize
         # store the psds and whiten the data
         self.psds = psds
 
-    @property
-    def high_frequency_cutoff(self):
-        """The high frequency cutoff of the inner product.
-
-        If a high frequency cutoff was not provided for a detector, it will
-        be ``None``.
-        """
-        return self._f_upper
-
-    @high_frequency_cutoff.setter
-    def high_frequency_cutoff(self, high_frequency_cutoff):
-        """Sets the high frequency cutoff.
-
-        Parameters
-        ----------
-        high_frequency_cutoff : dict
-            Dictionary mapping detector names to frequencies. If a high
-            frequency cutoff is not provided for one or more detectors, the
-            Nyquist frequency will be used for those detectors.
-        """
-        self._f_upper = {}
-        if high_frequency_cutoff is not None and bool(high_frequency_cutoff):
-            for det in self._data:
-                if det in high_frequency_cutoff:
-                    self._f_upper[det] = high_frequency_cutoff[det]
-                else:
-                    self._f_upper[det] = None
-        else:
-            for det in self._data.keys():
-                self._f_upper[det] = None
-
```

Comment:
Thanks for getting rid of the setter, it was needlessly complicated and didn't actually do anything useful. However, please keep `high_frequency_cutoff` as a property that returns `_f_upper` rather than an attribute. I don't want the user thinking they can change the high frequency cutoff by setting that. The reason this isn't settable after the init is the weights would need to be recalculated. By making it an attribute, the user might think they can change it by settting `high_frequency_cutoff`, when it actually doesn't do anything.

I just realized you had also changed `low_frequency_cutoff` from a property to an attribute in an earlier PR. Please switch that back too. So, change `self._f_lower = self.low_frequency_cutoff = low_frequency_cutoff` in the init to `self._f_lower = low_frequency_cutoff`. Then add:
```
@property 
def low_frequency_cutoff(self):
    """The low frequency cutoff of the inner product."""
    return self._f_lower
```
Ditto for `high_frequency_cutoff`.

Reply:
@cdcapano Ok, done.

Reply:
Thanks!